### DISCUSSION TITLE
Is it possible to use a /admin route instead of a /keystatic route?

### DISCUSSION THREAD

**Discussion by camdenvaughan** [Thinkmill/keystatic#1079] [General]
I was trying to poke around if this was possible to change but couldn't find where. Its a minor thing but I thought it could be useful to change, specifically when using with clients who don't know what keystatic is and I figure /admin is easier to remember.

**Comment by jjaimealeman** [comment]
I second this.

From a "security" perspective (as an example), I never liked how WordPress admin was always `wp-admin`.

**Comment by zanhk** [comment]
Linking thread on discord https://discordapp.com/channels/1139017922073866332/1139018073442103397/1215927431232356412

But yeah as said `/admin` is easier for customers to remember than `/keystatic`, you have to tell them to bookmark the page and sometimes the forget as well.

https://github.com/Thinkmill/keystatic/blob/41e54d3a9f123c81ce709744ff9431a960e2a3a4/packages/astro/src/index.ts#L48

**Comment by zanhk** [comment]
Right now I m using a redirect from `/admin` to `/keystatic`

But it would be nice it would be like that without any setup!

**Comment by davlet61** [comment]
a simple `options` param to be passed into the integration could be nice. something like:
```ts
keystatic({ adminUrl ... })

// Then dynamic pattern:
 pattern: `/${options.adminUrl}/[...params]`,  // default could remain `keystatic`
```
just a thought
